src/pytezos.py

```diff
@@ -0,0 +1,12 @@
+import pytezos
+from . import config
+
+
+assert config.SECRET_KEY is not None and len(config.SECRET_KEY) > 0, \
+  "Could not read secret key"
+
+admin_key = pytezos.pytezos.key.from_encoded_key(
+    config.SECRET_KEY
+)
+
+ptz = pytezos.pytezos.using(config.TEZOS_RPC, admin_key)
```

Comment:
I don't think this module is really warranted, as we're still depending on PyTezos in other places. I suggest merging it with tezos_manager (for instance in a single `tezos.py` or something).